lib/stdlib/src/base64.erl

```diff
@@ -219,38 +219,49 @@ mime_decode_binary(Result, <<0:8,T/bits>>) ->
     mime_decode_binary(Result, T);
 mime_decode_binary(Result0, <<C:8,T/bits>>) ->
     case element(C, ?DECODE_MAP) of
-	Bits when is_integer(Bits) ->
-	    mime_decode_binary(<<Result0/bits,Bits:6>>, T);
-	eq ->
-	    case tail_contains_more(T, false) of
-		{<<>>, Eq} ->
-		    %% No more valid data.
-		    case bit_size(Result0) rem 8 of
-			0 ->
-			    %% '====' is not uncommon.
-			    Result0;
-			4 when Eq ->
-			    %% enforce at least one more '=' only ignoring illegals and spacing
-			    Split =  byte_size(Result0) - 1,
-			    <<Result:Split/bytes,_:4>> = Result0,
-			    Result;
-			2 ->
-			    %% remove 2 bits
-			    Split =  byte_size(Result0) - 1,
-			    <<Result:Split/bytes,_:2>> = Result0,
-			    Result
-		    end;
-		{More, _} ->
-		    %% More valid data, skip the eq as invalid
-		    mime_decode_binary(Result0, More)
-	    end;
-	_ ->
-	    mime_decode_binary(Result0, T)
+    Bits when is_integer(Bits) ->
```

Comment:
My apologies. There appear to be tabs in the file that, in my editor, I have set at 4 spaces instead of the recommended 8, so I thought I was matching the file's coding style, but I was working from a bad starting point. I will address these issues. Thanks!